### DISCUSSION TITLE
visibility issues with some SVGs

### DISCUSSION THREAD

**Discussion by herzaso** [tailwindlabs/tailwindcss#13869] [Help]
Some SVGs have a problem when working with visibility as demonstrated in https://play.tailwindcss.com/b6q2tGNoCM
Notice - this is the same code and the only change is the replacements of the SVGs and adding `group-hover:visible`

If you hover over the first item (Customizing your tailwind.config.js file) you'll see the SVG, but on other items you probably won't

**Comment by adamwathan** [comment]
Hey! Your issue is that you have duplicate IDs for your clip paths. IDs need to be unique in HTML documents.

Here's a fixed version where each one has a unique ID:

https://play.tailwindcss.com/GYU6iCRL7N

**Reply by herzaso** [reply]
Thanks @adamwathan for the prompt response. I guess i'll have to find a way to make those IDs unique (I have a lot of icons but even if I go one by one, using the same icon twice in the same page won't work)

**Reply by hbendev** [reply]
If you're using svgo, there's a cleanupids setting you can play with.

**Comment by herzaso** [comment]
cleanupIds is on by default but it doesn't handle the same SVG being
rendered twice (I found inline-svg-unique-id plugin which works for me,
just wish it was out of the box)
